src/views/Alteration.vue

```diff
@@ -48,7 +46,16 @@
         @haveChanges="onAlterationSummaryChanges()"
       />
 
-      <no-fee-summary class="mt-10" />
+      <documents-delivery
+       class="mt-10"
+       :pleaseValidate="true"
```

Comment:
Does the red left border and red label apply to this component? If it does then you'll have to declare this prop and do something with it (ie, show error styling as applicable).

Reply:
it does not

Reply:
Then please delete unused code please.